Makefile

```diff
@@ -133,6 +133,10 @@ chplvis: compiler third-party-fltk FORCE
 	cd tools/chplvis && $(MAKE)
 	cd tools/chplvis && $(MAKE) install
 
+c2chapel: FORCE
+	cd tools/c2chapel && $(MAKE)
```

Comment:
I see that you've added docs on `make clean`, etc., but this comment was referring to the `$CHPL_HOME`-level `make` targets. Are you also holding off on documenting these until this is included in the release?